### PR TITLE
[FA-19] Home Graphics improvement

### PR DESCRIPTION
**Pull request by johnvamp**
image

### CODE REVIEW

**Comment by SantiguzmaN** [comment]
remember assign yourself the PR's and request the team review

File: src/components/home/homeContainer.jsx [hunk 1]
```diff
@@ -19,27 +19,42 @@ const Home = () => {
...
-          </h4>
-          <h4> NOMBRE DEL CONSULTORIO </h4>
-          <div className="border w-100">
-            {/* here must go the calendar */}
-          </div>
-          <button type="button" onClick={() => logOutFetch()}>
+    <div className="wrapper App">
+      <div className="row w-100 principal-Container">
+        <div className="border col-3" id="childenConteiner">
+          <h4 id="usercolor">{userName}</h4>
```
Comment (SantiguzmaN) [inline src/components/home/homeContainer.jsx]: why by id?? why not by className?

**Review by SantiguzmaN — CHANGES_REQUESTED** [review]
You must make changes

File: src/components/home/homeContainer.jsx (hunk 1 above)
Comment (johnvamp) [reply src/components/home/homeContainer.jsx]: at the beginning I had problems, the color of the component was deconfigured, that's why I had to do an id

**Review by skills20 — APPROVED** [review]

File: src/components/home/homeContainer.jsx (hunk 1 above)
Comment (johnvamp) [reply src/components/home/homeContainer.jsx]: in line 25 put an id, because they are attributes that only that line will have

**Review by SantiguzmaN — APPROVED** [review]

**Pull request merged**